tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+class ActionSelenium(argparse.Action):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    def __call__(self, parser, namespace, values, option_string=None):
+        browsers = values.split(',')
+        for browser in browsers:
+            try:
+                # Attempt to create class instance to see if everything is fine.
+                import_string('selenium.webdriver.%s.webdriver.WebDriver' % browser)
+            except Exception:
```

Comment:
I don't have a strong opinion on this. When it was like that before, I didn't really like the blank browser popping up before the tests started to run. If we change it though, we need to include the original exception in the error message that's reraised. I think the current simple approach is okay.